tmt/steps/provision/localhost.py

```diff
@@ -23,7 +23,7 @@ def _prepare_ansible(self, what):
         # Playbook paths should be relative to the metadata tree root
         playbook = os.path.join(self.step.plan.run.tree.root, what)
         # Run ansible against localhost, in verbose mode, enable --become
-        self.run(f'ansible-playbook -vb -c local -i localhost, {playbook}')
+        self.run(f'ansible-playbook -vb -c local -i localhost, {playbook}', log=self.verbose)
```

Comment:
Ah, I was gonna fix #48, but as you're touching it: We need to add `shell=True`. Without it we can't handle pipes, output redirection etc.

```
subprocess.Popen( . . . , shell=True)
```